pkgs/applications/networking/browsers/chromium/get-commit-message.py

```diff
@@ -13,7 +13,8 @@
 import feedparser
 import requests
 
-feed = feedparser.parse('https://chromereleases.googleblog.com/feeds/posts/default')
+# Official rss/atom feed taken from <https://chromereleases.googleblog.com/>'s html source (<link type="application/atom+xml">)
+feed = feedparser.parse('https://www.blogger.com/feeds/8982037438137564684/posts/default')
```

Comment:
Lets just hope Google in their infinite wisdom doesn't change this to some other URL soon :)